ci/build-rhoscala-rbls.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+source ci/header.sh
```

Comment:
I see what you're going for with `source ci/header.sh`, but you recognize that this is a pretty fragile way to include a file? There's no "search path" for this. I believe in bash if you put `ci` in `PATH`, then you can just `source header.sh` and it will find and run the `header.sh` file wherever it first appears in your $PATH. (See below excerpt from bash manual) That would make your top lines go something like this:

```
#!/usr/bin/env bash
export PATH=$PATH:$(pwd)/ci
source header.sh
```
Slightly less elegant, but more reliable and more interestingly customizeable, because now individual developers can put a `header.sh` in their ~/bin or something to over-ride settings for their local development environment.

Relevant section of bash man page:
```
       source filename [arguments]
              Read and execute commands from filename in the current shell environment and return
              the  exit  status of the last command executed from filename.  If filename does not
              contain a slash, filenames in PATH are used to find the directory containing  file‐
              name.   The  file searched for in PATH need not be executable.
```